ansible-scylla-node/tasks/adjust_keyspace_replication.yml

```diff
@@ -24,9 +24,19 @@
   loop: "{{ groups['scylla'] }}"
   run_once: true
 
+- set_fact:
+    scylla_admin_username: "{{ scylla_admin_default_user }}"
+    scylla_admin_password: "{{ lookup('ini', scylla_admin_default_user, section='cql_credentials', allow_no_value=true, file=inventory_file) }}"
+  when: "'cql_credentials' in groups"
+
+- set_fact:
+    scylla_admin_username: "cassandra"
+    scylla_admin_password: "cassandra"
+  when: scylla_admin_username is not defined or scylla_admin_password is not defined or scylla_admin_password == ''
```

Comment:
The last branch is not clear. AFAIR empty password is a legal password, isn't it?

Reply:
![image](https://user-images.githubusercontent.com/6103981/227565716-5447c86f-c099-4120-a8de-bb0d65c24724.png)

Reply:
Fixed